Administrator Guide/NFS-Ganesha GlusterFS Integration.md

```diff
@@ -119,6 +121,9 @@ Step 4 can be performed via following script
        - *#dbus-send --type=method_call --print-reply --system --dest=org.ganesha.nfsd /org/ganesha/nfsd/ClientMgr org.ganesha.nfsd.clientmgr.ShowClients*
 
 ## Using Highly Available Active-Active NFS-Ganesha And GlusterFS cli
+
+### Please Note currently HA solution for nfs-ganesha is only available in 3.10 From 3.12 onwards HA will be handled by different project known as [storhaug](https://github.com/linux-ha-storage/storhaug) which is under development.
```

Comment:
Please add punctuation to this sentence and do not use a markdown heading. You can use a [markdown Blockquotes](https://daringfireball.net/projects/markdown/syntax#blockquote) for this.

Reply:
Done